### ISSUE TITLE
Windows Patch Search Accessibility

### ISSUE THREAD

**Issue by baconpaul** [surge-synthesizer/surge#6408]
Moved here from #5714 

user report from pietermach:

> The typeahead list works great on Mac, if I downarrow into the results VO reads them beautifully.  On Windows, when I downarrow into the list, the screen readers think that the list itself is focused and not one of the items and so nothing apart from "list" is spoken. Also, if I move through the objects with screen reader keys, On Mac the rows just show up as the patch names which is great, while on Windows the cells are first labeled as "row X" and only if you go inside them you can see the label. Don't know if this is why Windows doesn't pick up the focus but it could be a clue maybe?

FOr a couple of reasons I've been unable to fix this so I added the workaround that windows will use the announcer channel to announce the focused item. This is obviously insufficient but lets us ship 1.1.0 and put this into the 1.1.n milestone.

**Comment by mkruselj** [comment]
This issue is the same as this one: https://forum.juce.com/t/listbox-accessibility-doesnt-work-on-windows/53433/

So I'm closing it as JUCE crew has already been notified.
